4-js/index.js

```diff
@@ -0,0 +1,330 @@
+                    resultSort += 'address_like=' + addressInput.value + '&&';
+                }
+                if (e.keyCode === 8&&companyInput.value === ''&&nameInput.value === ''&&addressInput.value === '') {
+                  getUsersInfo('_start=' + (paginations.length * 10 - getCurrentNumberPogination()) + '&_limit=50');
+                } else {
+                  getUsersInfo(resultSort);
+                }
+            }
+        })
+    }
+}
+
+function getCurrentNumberPogination() {
+  let el = document.querySelector('.active');
+  return el.name;
+}
+
+function createUser() {
+  let newUser = {
+    company: userCompany.value,
+    name: userName.value,
+    address: userAddress.value,
+    city: userCity.value,
+    country: userCountry.value,
+  };
+  sendRequsest('POST', requestUrl, newUser)
+    .then((data) => {
+      createTableUsers([data],'dont-delete');
+    })
+    .catch((err) => console.log(err));
+}
+
+function formCheck() {
+  if (
+    userCompany.value !== '' &&
+    userName.value !== '' &&
+    userAddress.value !== '' &&
+    userCity.value !== '' &&
+    userCountry.value !== ''
+  ) {
+    createUserBtn.removeAttribute('disabled', 'disabled');
+    createUserBtn.classList.remove('is-translucent');
+  } else {
+    createUserBtn.classList.add('is-translucent');
+  }
+}
+
+function getUsersInfo(arg) {
+  sendRequsest('GET', requestUrl + '?' + arg)
+    .then((data) => createTableUsers(data))
+    .catch((err) => console.log(err));
+}
+
+function createTableUsers(arr, check) {
+  const tableUser = document.getElementsByClassName('table-user')[0];
+  if (check !== 'dont-delete') {
+    tableUser.innerHTML = '';
+  }
+  for (let i = 0; i < arr.length; i++) {
+    const tr = document.createElement('tr');
+    tr.id = arr[i].id;
+    tr.classList.add('row');
+
+    const tdCheckbox = document.createElement('td');
+    const checkbox = document.createElement('input');
+    const label = document.createElement('label');
+    const span = document.createElement('span');
+    checkbox.classList.add('custom-input');
+    checkbox.classList.add('user-checkbox');
+    checkbox.name = arr[i].id;
+    span.classList.add('custom-span');
+    label.classList.add('custom-checkbox');
+    checkbox.type = 'checkbox';
+    label.appendChild(checkbox);
+    label.appendChild(span);
+    tdCheckbox.appendChild(label);
+    tr.appendChild(tdCheckbox);
+
+    createTdElement(arr[i].company, tr);
+    createTdElement(arr[i].name, tr);
+    createTdElement(arr[i].address, tr);
+    createTdElement(arr[i].city, tr);
+    createTdElement(arr[i].country, tr);
+    createTdButtonElement(arr[i].id, tr, 'btn-edit');
+    createTdButtonElement(arr[i].id, tr, 'btn-delete');
+
+    tableUser.insertBefore(tr, tableUser.firstChild);
+  }
+}
+
+function createTdElement(value, parent) {
+  const td = document.createElement('td');
+  td.innerText = value;
+  parent.appendChild(td);
+}
+
+function createTdButtonElement(value, parent, style) {
```

Comment:
instead of `style` I would call as `mode` parameter it is more obviosly. (edit mode, create mode etc.)

Reply:
renamed